include/galahad_arc.h

```diff
@@ -995,7 +995,7 @@ void arc_solve_without_mat( void **data,
  @param eval_f is a user-supplied function that must have the following
    signature:
    \code
-        int eval_f( int n, const double x[], double *f, const void *userdata )
```

Comment:
@nimgould, I noticed that you use `double` in some function signatures.
Did you forget to replace them by `real_wp_`, or is it intentional?
Should I just use `int` in this context, or is keeping `ipc_` acceptable?

Reply:
Yes, I think this was only for the docs (I do the same in the docs/C/rst files). This should be changed at some stage. I believe that the codes themselves use real_wp_. All of the docs need to replace int with ipc_ as well., these all predate the 64 bit extension.